lib/ansible/plugins/filter/regex_findall.yml

```diff
@@ -3,22 +3,22 @@ DOCUMENTATION:
   version_added: "2.0"
   short_description: extract all regex matches from string
   description:
-    - Search in a string or extract all the parts of a string matching a regular expression
+    - Search in a string or extract all the parts of a string matching a regular expression.
   positional: _input, _regex
   options:
     _input:
-      description: String to match against
+      description: String to match against.
       type: str
       required: true
     _regex:
-      description: regular expression string that defines the match
+      description: Regular expression string that defines the match.
       type: str
     multiline:
-      description: search across line endings if C(True), don't if otherwise
+      description: Search across line endings if C(True), do not if otherwise.
```

Comment:
```suggestion
      description: Search across line endings if C(True). Stop searching at first line ending if C(False).
```

Reply:
That's not correct. It basically searches inside lines (and not just the first line) when this is `False`.

Reply:
Okay, I don't really understand what "not searching across line endings" means, then. Does this only come into play when searching a phrase? 

Anyway, I withdraw the suggestion, thanks for correcting me!

Reply:
carriage return, really the hidden symbol \n which signifies 'go to next line' (or \r\n on windows).